overlord/devicestate/devicestate.go

```diff
@@ -945,6 +1035,8 @@ func Remodel(st *state.State, new *asserts.Model) (*state.Change, error) {
 
 		prepare := st.NewTask("prepare-remodeling", i18n.G("Prepare remodeling"))
 		prepare.WaitFor(requestSerial)
+		prepare.Set("local-snaps", localSnaps)
+		prepare.Set("local-paths", paths)
```

Comment:
we probably should error here for now instead of supporting this case, as rereg needs networking, which also means we don't need the change in handlers_remodel.go